cranelift/codegen/src/isa/aarch64/lower.isle

```diff
@@ -99,6 +99,14 @@
           (add_with_flags_paired $I64 x_lo y_lo)
           (adc_paired $I64 x_hi y_hi))))
 
+;;;; Rules for `scalar_to_vector` ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(rule (lower (has_type (multi_lane 64 _) (scalar_to_vector x)))
+      (mov_to_fpu x (ScalarSize.Size64)))
+
+(rule (lower (has_type (multi_lane _ _) (scalar_to_vector x)))
```

Comment:
Here as well we'll want a rule priority (as in #4399) so this rule is ensured to be matched after the above case.